base_report_to_printer/models/ir_actions_report.py

```diff
@@ -56,12 +56,16 @@ def print_action_for_report_name(self, report_name):
         }
         return serializable_result
 
+    def _get_user_default_printer(self, user):
+        return user.printing_printer_id
```

Comment:
```suggestion
        self.ensure_one()
        return user.printing_printer_id
```
As we will check `self.label` in the override.

Reply:
@ne1800 ping